core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.spec.ts

```diff
@@ -993,11 +1007,15 @@ describe('Settings Tab Component', () => {
     });
 
     it('should refresh settings tab when refreshSettingsTab event occurs',
-      () => {
+      fakeAsync(() => {
```

Comment:
Why are we testing `RouterService.onRefreshSettingsTab.subscribe(` twice?

1. https://github.com/oppia/oppia/blob/dda91385010ac80df64f7b8d967cc0719d31753d/core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.spec.ts#L1009-L1018
2. https://github.com/oppia/oppia/blob/dda91385010ac80df64f7b8d967cc0719d31753d/core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.spec.ts#L283-L293

Reply:
I am not sure why that is, to be honest.

Reply:
Could you look into the difference and remove the unnecessary test?

Reply:
Done, I've merged both tests into one. PTAL.